doc/confidence.rst

```diff
@@ -198,22 +203,30 @@ using :func:`conf_interval2d`:
     axes[0, 0].legend()
 
     aix, aiy = 0, 0
-    nsamples = 30
+    nsamples = 50
+    explicitly_calculate_sigma = True
+    
     for pairs in (('sigma', 'amplitude'), ('intercept', 'amplitude'),
                   ('slope', 'intercept'), ('slope', 'center'), ('sigma', 'center')):
+    
         xpar, ypar = pairs
-        print("Generating chi-square map for ", pairs)
-        c_x, c_y, dchi2_mat = conf_interval2d(out, out, xpar, ypar,
-                                              nsamples, nsamples,
-                                              nsigma=3.5, chi2_out=True)
-        # sigma matrix: sigma increases chi_square
-        # from  chi_square_best
-        # to    chi_square + sigma**2 * reduced_chi_square
-        # so:   sigma = sqrt(dchi2 / reduced_chi_square)
-        sigma_mat = np.sqrt(abs(dchi2_mat)/out.redchi)
-
-        # you could calculate the matrix of probabilities from sigma as:
-        # prob_mat  = np.erf(sigma_mat/np.sqrt(2))
+        if explicitly_calculate_sigma:
+            print("Generating chi-square map for ", pairs)
```

Comment:
please use a formatting string

Reply:
@newville @reneeotten Ok thanks, I'll check and fix it.